src/Components/Authentication/Desktop/ModalManager.tsx

```diff
@@ -14,6 +16,9 @@ export interface ModalManagerProps {
   submitUrls: { [P in ModalType]: string }
   csrf: string
   redirectUrl?: string
+  tracking?: any
+  trackingData?: any
```

Comment:
@kanaabe could you define the type for `trackingData`? Would be helpful to look at a glance what's being logged